distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -553,26 +554,230 @@ type Column
+    ## PRIVATE
+
+       Creates a column from a single constant value. The value must be of a
+       type supported by the database back-end.
+
+       This method is not static; it must be called on an existing column. That
+       existing column is only used for its internal state; the contents of the
+       column are not used.
+
+       Arguments:
+       - value: the constant value to use for the entire column. The value must
+         be of a type supported by the database back-end.
+
+       > Example
+         Create a column of the value 42
+
+         column.const 42
+    const : Any -> Column
+    const self value = self.to_table.make_constant_column value
+
+    ## Round to a specified number of decimal places.
+
+       `round` is only valid for `Value_Type.Decimal`, `Value_Type.Float`, and
+       `Value_Type.Integer` columns.
+
+       By default, rounding uses "symmetric round-half-up", also known as
+       "round towards 0." If use_bankers=True, then it uses "round-half-even",
+       also known as "banker's rounding".
+
+       The return type depends on the backend.
+
+       Arguments:
+       - decimal_places: The number of decimal places to round to. Can be
+         negative, which results in rounding to positive integer powers of 10.
+         Must be between -15 and 15 (inclusive).
+       - use_bankers: Rounds mid-point to nearest even number.
+
+       ! NaN/Inf
+         If a `NaN` or `Inf` value is passed to `round`, it immediately returns
+         the same value.
+
+       ! Error Conditions
+
+         If `decimal_places` is outside the range -15..15 (inclusive), an
+         `Illegal_Argument` error is thrown.
+
+       ? Negative decimal place counts
+         Rounding to `n` digits can be thought of as "rounding to the nearest
+         multiple of 10^(-n)". For negative decimal counts, this results in
+         rounding to the nearest positive integer power of 10.
+
+       > Example
+         Round a column to two decimal places.
+
+             import Standard.Examples
+
+             example_round = Examples.decimal_column.round 2
+    round : Integer -> Boolean -> Column | Illegal_Argument | Invalid_Value_Type
+    round self decimal_places=0 use_bankers=False =
+        Rounding_Helpers.check_decimal_places decimal_places <| Value_Type.expect_numeric self <|
+            if self.should_use_builtin_round decimal_places use_bankers then self.round_builtin decimal_places else
+                self.short_circuit_special_floating_point <|
+                    case self.value_type of
+                        Value_Type.Integer _ -> self.round_integer decimal_places use_bankers
+                        Value_Type.Float _ -> self.round_float decimal_places use_bankers
+                        Value_Type.Decimal _ _ -> self.round_decimal decimal_places use_bankers
+                        _ ->
+                            expected = "Value_Type.Decimal, Value_Type.Float, or Value_Type.Integer"
```

Comment:
just a suggestion, but I'm not sure if we need the prefix here? the error is about Value_Types already.

```suggestion
                            expected = "Decimal, Float, or Integer"
```

Reply:
Done